packages/edit-post/src/components/layout/index.js

```diff
@@ -220,13 +222,47 @@ function MetaBoxesMain( { isLegacy } ) {
 	const separatorRef = useRef();
 	const separatorHelpId = useId();
 
-	const [ isUntouched, setIsUntouched ] = useState( true );
+	const heightRef = useRef();
+	const actualizeHeight = ( candidateHeight, isPersistent ) => {
```

Comment:
I think you mean "update" ("make (something) more modern or up to date") instead of "actualize" ("make a reality of").

Reply:
I guess "actualize" was already used in the code before this PR. Still, might be a good chance to fix it as the name does not match what it actually does, afaik.

Reply:
I’m okay with "update". I bet no one would have given it second glance so perhaps that alone makes it better.

I’m always interested in words and using them better so I’ll take the opportunity to ask your thoughts on why what this function does is not an actualization.

My own convention with  "update" is to use it for functions that modify the value relatively or take a callback which receives the current value and uses the returned value to set the new one.

As for "actualize", to me it signifies effects. In this case the function may apply two DOM properties. Which after having another look at the other "actualize" in the file, I want to say that was mistaken (and my doing). It only sets state so I should have named it something else 😆.

Maybe instead of "actualize" there is a better word—"affect" or "apply"? Just wondering and not specifically for this case. "updateHeight" is fine by me.

Reply:
I'm missing the context here, but my 2cents are that 1. "actualize" just sounds odd to me in this context and 2. "update" instantly hint to something being up-to-date (in this case it seems to imply "keeping DOM properties up-to-date" from what I understood of your response) so it seems okay. FWIW, "apply" also sounds okay to me - and if more accurate, then that'd be my preference over "update". "affect" feels weird to me.

Also, this immediately stood out to me because many native spanish speakers tend to use that word when they really mean "update", because update is "actualizar" in Spanish so it's a very common ["false friend"](https://en.wikipedia.org/wiki/False_friend).

In any case, I don't have the context, so this is a very optional nit based on a very quick and shallow look at the code, and I'm personally okay with any choice you make here :)

Reply:
Btw, I don't disagree "actualize" is a technically correct description of what the function does. It's just not a very accessible word, especially to non-native speakers, and it's not that commonly used. I think it's the first time I've personally seen it outside of a philosophy or psychology context (or a fellow native Spanish speaker wrongly using it in place of "update"). So that's what I mean by odd, no more no less.

I don't know if you chose the original naming - FWIW I think if intentional, it's an interesting choice that can lead to some interesting linguistic conversations, however my points here are purely rooted in practicality :P

Reply:
Thanks for elaborating. I concur "actualize" is uncommon enough that it may fail to add any clarity. So I’ve renamed the functions that were using it b1b0ad2ec75640ae9c2b244724ea08fc7addd8cf. It is  about the false friend status of actualize and actualizar. I would have to admit my willingness to use "actualize" is influenced by Spanish as a second language though not in place of "update". Merely that I think the word might be underused in English.